### DISCUSSION TITLE
Is it possible to have a warning when code in a computed property attempts to modify an observable value?

### DISCUSSION THREAD

**Discussion by szagi3891** [mobxjs/mobx#3808] [Ideas]
Is it possible to configure MobX to issue a warning when code in a computed property tries to modify an observable value?

**Comment by domehead100** [comment]
I’m 95% sure that Mobx already does that and has for at least seven or more years.

It is possible to modify non-observable values if I recall, or to do whatever you want asynchronously like in a setTimeout since the tracking is done synchronously (though not recommending that).
